### PR TITLE
implemented dsep unified flow

### PR DESCRIPTION
**Pull request by kundanEminds**

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -211,3 +211,6 @@ iframe .header-container .location-btn {
 .card-sub-containers li:hover {
   scale: 1.03;
 }
+/* iframe::-webkit-scrollbar {
```
Comment (ujjwal502) [inline src/App.css:214]: Please remove this commented code if not required

File: src/common/Tabs/tabs.tsx [hunk 2]
```diff
@@ -25,6 +25,7 @@ const TabsComponent: React.FC<TabsComponentPropsModal> = ({
   const OSC = localStorage.getItem("name") === "OSC";
   const DSEP = localStorage.getItem("name") === "DSEP";
   const UEI = localStorage.getItem("name") === "UEI";
+  const dsepUnified = localStorage.getItem("name") === "dsepUnified";
```
Comment (ujjwal502) [inline src/common/Tabs/tabs.tsx:28]: change this variable name to isDsepUnified

**Review by ujjwal502 — DISMISSED** [review]

File: src/App.css (hunk 1 above)
Comment (kundanEminds) [reply src/App.css]: done

File: src/common/Tabs/tabs.tsx (hunk 2 above)
Comment (kundanEminds) [reply src/common/Tabs/tabs.tsx]: Done

**Pull request merged**